cmd/init.go

```diff
@@ -179,6 +179,13 @@ func init() {
 		defaultDashboardVersion = dashboardVersionEnv
 	}
 
+	defaultContainerRuntime := "docker"
```

Comment:
Constant present in utils/utils.go might be used.
Normally we try to prefix env variables with `dapr`. So can we keep something like `dapr_container_runtime` ?

Reply:
@pravinpushkar you mean the local variable name which is now called `defaultContainerRuntime` should be renamed? Because the env var taken into context is called `DAPR_CONTAINER_RUNTIME`. 

I'll update the PR to pull "docker" from `utils/utils.go`